app/models/katello/content_view_history.rb

```diff
@@ -66,5 +66,13 @@ def humanized_status
         _("Success")
       end
     end
+
+    def description
+      if self.task.try(:label) == "Actions::Katello::ContentView::Publish"
+        content_view_version.description
```

Comment:
That's a good question and something I wondered about myself while working on this. It looks like I added the description field to versions shortly after cv histories were created so I'm guessing I didn't realize that cv histories had notes. I could open an issue or fix it here?